content/en/_index.md

```diff
@@ -65,6 +65,30 @@ Use [W&B Inference]({{< relref "/guides/inference/" >}}) to access leading open-
 
 </div>{{% /card %}}
 {{< /cardpane >}}
+
+</div>
+
+<div class="bottom-row-cards">
+{{< cardpane >}}
+{{% card %}}<div onclick="window.location.href='/guides/training/'" style="cursor: pointer;">
+
+<div className="card-banner-icon" style="float:left;margin-right:10px !important; margin-top: -12px !important">
+{{< img src="/icons/Name=Training, Mode=Dark.svg" width="60" height="60" >}}
+</div>
+<h2>W&B Training</h2>
+
+### Post-train your models
+
+Now in public preview, use [W&B Training]({{< relref "/guides/training/" >}}) to post-train large language models using serverless reinforcement learning (RL). Features include fully managed GPU infrastructure, integration with ART and RULER, and automatic scaling for multi-turn agentic tasks.
+
+- [Introduction]({{< relref "/guides/training/" >}})
+- [Serverless RL]({{< relref "/guides/training/serverless-rl/" >}})
+- [Prerequisites]({{< relref "/guides/training/prerequisites/" >}})
+- [API Reference]({{< relref "/guides/training/api-reference/" >}})
+
+</div>{{% /card %}}
+{{< /cardpane >}}
```

Comment:
Might need some CSS here. The width of the card stretches across (see attached image)
<img width="1727" height="980" alt="Screenshot 2025-10-06 at 4 44 28 PM" src="https://github.com/user-attachments/assets/e6ad4558-10e1-4c01-a5fd-0999307630b0" />